### ISSUE TITLE
(Bug report) Router not found for request /assets/v0.60.3/app-dist/desktop.js

### ISSUE THREAD

**Issue by JenYiet** [zadam/trilium#4040]
### Trilium Version

0.60.3

### What operating system are you using?

Other Linux

### What is your setup?

Server access only

### Operating System Version

Debian GNU/Linux 10

### Description

After Logging in I get this blank Page.

![image](https://github.com/zadam/trilium/assets/69795260/08819223-5992-4489-a517-37f621ba2cdb)

### **Backend Log:**
16:22:21.793 App HTTPS server starting up at port 8080
16:22:21.794 {
  "appVersion": "0.60.3",
  "dbVersion": 213,
  "syncVersion": 29,
  "buildDate": "2023-06-15T23:23:37+02:00",
  "buildRevision": "3223e767875e5379c99ff58a562cb9c1a2641bdf",
  "dataDirectory": "/root/.local/share/trilium-data",
  "clipperProtocolVersion": "1.0",
  "utcDateTime": "2023-06-18T16:22:20.427Z"
}
16:22:21.798 Listening on port 8080
16:22:21.847 Becca (note cache) load took 47ms
16:22:25.857 Table counts: notes: 471, note_revisions: 255, branches: 511, attributes: 714, etapi_tokens: 0
16:22:26.244 Slow query took 357ms: SELECT notes.noteId, notes.type, notes.mime FROM notes JOIN note_contents USING (noteId) WHERE isDeleted = 0 AND isProtected = 0 AND content IS NULL
16:22:26.266 All consistency checks passed with no errors detected (took 410ms)
16:25:35.720 Generated CSRF token mxrB4aOR-saUK8FXED3O3_3DvlvXS950mql8 with secret _csrf=yzi5E33-4jbrenYhOiwssN3J; Path=/
16:25:35.977 Error: Router not found for request /assets/v0.60.3/app-dist/desktop.js
16:27:12.215 Generated CSRF token tOE4LqtY-gYhB_Fkr2vvulFfWIuUwV3WRgAY with secret undefined
16:27:12.284 Error: Router not found for request /assets/v0.60.3/app-dist/desktop.js
16:27:13.077 Generated CSRF token OqIY3gU3-uJj5rrqL9ZVi2vgg8fEFig9McKE with secret undefined
16:27:13.144 Error: Router not found for request /assets/v0.60.3/app-dist/desktop.js
16:27:13.652 Generated CSRF token RQMLb0oA-x6cSxtrJlo2URz7Is51PLzDfXdE with secret undefined
16:27:13.706 Error: Router not found for request /assets/v0.60.3/app-dist/desktop.js
16:27:14.118 Generated CSRF token bmUeLTZ8-9_s3G-pD1QzgPdvF_ilwHm9tqzs with secret undefined
16:27:14.179 Error: Router not found for request /assets/v0.60.3/app-dist/desktop.js
16:27:14.530 Generated CSRF token iNBHjXmk-CYkO2RPdl2egmtfqMjj31WeifG0 with secret undefined
16:27:14.578 Error: Router not found for request /assets/v0.60.3/app-dist/desktop.js
16:28:49.791 Generated CSRF token 83Hp31d4-xqW7UnNPtUA6Lql06ERCTwY66gE with secret _csrf=oEhv6TVsbi99aw0ZW5VdH4K8; Path=/
16:28:49.990 Error: Router not found for request /assets/v0.60.3/app-dist/desktop.js

### **Fronted Log:**
On Chrome:
![Screenshot 2023-06-18 184246](https://github.com/zadam/trilium/assets/69795260/d89d3c5f-471b-4d76-b987-68931b68dc3a)

On Firefox:
![image](https://github.com/zadam/trilium/assets/69795260/851b8c82-e68d-4fc1-8e67-e7cf28458bc6)

After Deleting the Data-Directory, restarting Trilium on the server, deleting Cache and Cookies on the browser:
![image](https://github.com/zadam/trilium/assets/69795260/7d049785-9dc4-4c09-b5d4-9a328f929353)

**Comment by zadam** [comment]
Am I right to assume that you run from git checkout? In that case you likely did not build the webpack assets.

This way of running trilium is not well supported, it's recommended to use a build from https://github.com/zadam/trilium/releases/latest

**Comment by JenYiet** [comment]
I encountered some issues after updating from version 0.56.2 to 0.60.3 of Trilium. Initially, when I ran Trilium using the command "node src/www," the problem persisted. However, I managed to resolve it by adding the command "TRILIUM_ENV=dev node src/www"

Unfortunately, another issue arose with the Trilium data from version 0.56.2. Several images were not appearing, and some resources could not be found. For instance, the logo was missing. However, when I moved the Trilium data directory to a different location and restarted Trilium, these issues disappeared. Upon returning the old Trilium data to its original location, the problem reappeared once again.

**Comment by zadam** [comment]
@JenYiet just to make sure, you downloaded and used [trilium-linux-x64-0.60.3.tar.xz](https://github.com/zadam/trilium/releases/download/v0.60.3/trilium-linux-x64-0.60.3.tar.xz) ? Because such build should not have these problems ... Or did you perhaps used [Source code (tar.gz)](https://github.com/zadam/trilium/archive/refs/tags/v0.60.3.tar.gz) ?

**Comment by zadam** [comment]
@JenYiet just to make sure, you downloaded and used [trilium-linux-x64-0.60.3.tar.xz](https://github.com/zadam/trilium/releases/download/v0.60.3/trilium-linux-x64-0.60.3.tar.xz) ? Because such build should not have these problems ... Or did you perhaps used [Source code (tar.gz)](https://github.com/zadam/trilium/archive/refs/tags/v0.60.3.tar.gz) ?

**Comment by JenYiet** [comment]
I used Source Code. I followed the Instructions here: https://github.com/zadam/trilium/wiki/Manual-server-installation
When I did it about 6 - 8 months ago with Version 0.56.02. I had no issues. I updated to version 0.60.03 following the instructions.  
I had problems when I did not add the TRILIUM_ENV=dev part. After adding most problems gone. Only My Data Directory is making problems. When I move the Data Directory to another location and restart trilium then all problems are gone. But this is not an option. I need the data.
